go.sum

```diff
@@ -11,6 +11,8 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/dynatrace/dynatrace-configuration-as-code-core v0.7.1-0.20250207115859-d33243b1fdf6 h1:F3cTkffc2xH6IK/MS3gPzNgpjcxG71ik72dTrfnv9ig=
 github.com/dynatrace/dynatrace-configuration-as-code-core v0.7.1-0.20250207115859-d33243b1fdf6/go.mod h1:QUFQtBjb6BkrzqBzCisqg4c1olv2FWC3rhYnxAyQoRo=
+github.com/dynatrace/dynatrace-configuration-as-code-core v0.7.1-0.20250211150749-1391cd8fbd8b h1:uUspeVi3gCtcttfWEOnwnkN9PAoDfDbnNwjOMrv2E0M=
```

Comment:
After updating the project with a newer version, run `go mod tidy` to remove reference to the previous version (ln. 12-13)

Reply:
Noted, thanks.
Rebased to main, therefore the change is not needed anymore